### ISSUE TITLE
`struct mifare_classic_block_manufacturer` member `btUnknown` should be `btSAK`

### ISSUE THREAD

**Issue by GoogleCodeExporter** [timmerk/libnfc#250]
```
I tested changing this field on "Chinese magic cards" (S50)
It shows that this field is exactly the same as SAK returned by PICC
```

Original issue reported on code.google.com by `zhangyo...@gmail.com` on 4 Jun 2013 at 6:49

**Comment by GoogleCodeExporter** [comment]
```
We will to that change but beware is not exactly equal to SAK systematically:
* NXP MFC 1k: SAK=08 btSAK=88
* NXP MFC 4k: SAK=18 btSAK=98
* NXP MFC mini: SAK=09 btSAK=89
* NXP MFC 7bytes: SAK=08 btSAK=6th byte of UID
* NXP MF+ SL1: SAK=18 btSAK=28
* Infineon MFC: SAK=88 btSAK=88
* JCOP/SMartMX: SAK=38 btSAK=38
So
* 6 LSBits are = SAK
* 8th bit (MSB) is for sure not part of SAK
* 7th bit: no idea 

BTW abtUnknown can be labeled abtManuf as it contains the so called 
manufacturer data.
```

Original comment by `yob...@gmail.com` on 12 Jun 2013 at 5:52
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 7638e9ea87bf.
```

Original comment by `yob...@gmail.com` on 13 Jun 2013 at 6:17
- Changed state: **Fixed**
